src/sql/workbench/contrib/query/browser/queryActions.ts

```diff
@@ -234,11 +228,12 @@ export class RunQueryAction extends QueryTaskbarAction {
 		return;
 	}
 
-	public runQuery(editor: QueryEditor, runCurrentStatement: boolean = false) {
+	private async runQuery(editor: QueryEditor, runCurrentStatement: boolean = false): Promise<boolean> {
```

Comment:
Not sure why this function is `public`, marking it private doesn't seem to cause any issue. Let me know if there's any concern here.